include/miroil/miroil/edid.h

```diff
@@ -34,31 +34,13 @@ struct Edid
     PhysicalSizeMM size{0,0};
 
     struct Descriptor {
-        enum class Type : uint8_t {
-            timing_identifiers = 0xfa,
-            white_point_data = 0xfb,
-            monitor_name = 0xfc,
-            monitor_limits = 0xfd,
-            unspecified_text = 0xfe,
-            serial_number = 0xff,
-
-            undefined = 0x00,
-        };
-
-        union Value {
-            char monitor_name[13];
-            char unspecified_text[13];
-            char serial_number[13];
-        };
-
-        Type type{Type::undefined};
-        Value value{{0}};
-
-        std::string string_value() const;
+        std::string string_value() const { return {}; };
     };
-    Descriptor descriptors[4];
 
-    Edid& parse_data(std::vector<uint8_t> const&);
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wpedantic"
+    Descriptor descriptors[0];
```

Comment:
I get that this allows Lomiri to build - but this could break things at runtime. Have we checked?

Reply:
You're not wrong…

```cpp
    for (int i = 0; i < 4; i++) {
        str += QString::fromStdString(edid.descriptors[i].string_value());
    }
```

Reply:
Let's see how they react:

https://gitlab.com/ubports/development/core/lomiri/-/merge_requests/237

Reply:
:peanuts: